tests/e2e/features/createRecipe.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Create recipe feature
+    As a user
+    I want to create a recipe
+    So that I can store and share with other users
+
+    Background:
+        Given the superuser has signed up with the following details
+            | username         | superuser |
+            | password         | super1234 |
+        And the superuser has logged in with username "superuser" and password "super1234 "
+        And the superuser has created a space named "Superuser's Space"
+
+
+    Scenario: superuser creates a new recipe with basic details only
+        When the superuser creates new recipe "Masala tea recipe(test)"
+        And the superuser adds the following recipe details
+            | Description      | This is a test recipe for masala tea |
+            | Preparation_time | 2                                    |
+            | Waiting_time     | 10                                   |
+            | Servings         | 4                                    |
+            | Servings_text    | cup                                  |
+            | Image            | masalachai.jpeg                      |
+        And the superuser "Save & View" the recipe
+        Then the superuser should be redirected to the "Masala tea recipe(test)" recipe page
+
+
+    Scenario: superuser creates a new recipe with a step
+        When the superuser creates new recipe "Masala tea recipe(test)"
+        And the superuser adds the following recipe details
```

Comment:
is the `recipe details` a mandatory one when creating a recipe or there are no mandatory fields?
if this one's needed everytime then maybe the step can be defined as
```feature
 When the superuser creates a new recipe "Masala tea recipe(test)" with the following recipe details
```

and add other definitions for step details - ingredients

otherwise what you're doing is okay. I didn't notice there are different steps to add recipe in previous review